codex-ui/src/styles/themes/classic.pcss

```diff
@@ -0,0 +1,44 @@
+body[theme-base='classic'],
+body[theme-accent='classic'] {
+  /**
+  * Classic Dark Colors
+  */
+  --classic--bg-primary: #000000;
```

Comment:
why not `--classic-bg-primary`?

Reply:
to better separate theme name from variable name. This variable won't be used directly, `--base--bg-primary` or `--accent--bg-primary` will be used in components

Reply:
very error-prone decision, in my opinion

Reply:
its like a modifier, added to the beginning. `--base-text` and `--base--text` have different meaning